charts/sonarqube-dce/Chart.yaml

```diff
@@ -25,13 +25,7 @@ maintainers:
 annotations:
   artifacthub.io/changes: |
     - kind: changed
-      description: "Update SonarQube to 9.9.0"
-    - kind: changed
-      description: "Bootstrap chart version 7.x.x dedicated to SonarQube 9.9 LTS"
-    - kind: changed
-      description: "Update the list of supported kubernetes versions"
-    - kind: changed
-      description: "Set a new default (maximum) allowed size of the client request body on the ingress"
+      description: "Bootstrap chart version 8.x.x+ dedicated to future SonarQube non-LTS releases"
```

Comment:
We could mention two changes:
- kind: change desc: "Update SonarQube logo"
- kind: change desc: "Bootstrap chart version 8.x.x+ dedicated to the future SonarQube 10.0"